proto/cosmos/crypto/multisig/v1/keys.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package cosmos.crypto.multisig.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keys/multisig";
+
+// LegacyAminoMultisigThresholdPubKey specifies a public key type 
+// which nests multiple public keys and a threshold,
+// it uses legacy amino address rules
+message LegacyAminoMultisigThresholdPubKey {
+    option (gogoproto.goproto_getters) = false;
+
+    uint32 threshold = 1 [(gogoproto.moretags) = "yaml:\"threshold\""];
+    repeated google.protobuf.Any public_keys = 2 [(gogoproto.customname) = "PubKeys", (gogoproto.moretags) = "yaml:\"pubkeys\""];
+}
```

Comment:
Why are these in separate proto packages? That just increases the type URL length. I would prefer something short like `cosmos.crypto.MultisigPubKey` or `cosmos.crypto.multisig.PubKey` instead of `cosmos.crypto.multisig.v1. LegacyAminoMultisigThresholdPubKey`

Reply:
That seemed cleaner to me to implement their methods in two separate packages but I agree we could make the multisig pubkey name shorter, `cosmos.crypto.multisig.PubKey` sounds good to me, although we might wanna keep the `LegacyAmino` prefix?

Reply:
It's now:
- `cosmos.crypto.multisig.LegacyAminoPubKey`
- `cosmos.crypto.secp256k1.{Pub,Priv}Key`